README.md

```diff
@@ -1,94 +1,157 @@
+Note that the `tests/test_charm.py` file imports `ops.testing` and uses it to set up a testing harness:
 
-```python
+```
+import ops.testing
+...
 class TestCharm(unittest.TestCase):
-    def test_foo(self):
-        harness = Harness(MyCharm)
-        self.addCleanup(harness.cleanup)  # always clean up after ourselves
+    def setUp(self):
+        self.harness = ops.testing.Harness(MyNewCharmCharm)
+        self.addCleanup(self.harness.cleanup)
+        self.harness.begin()
+
+    def test_httpbin_pebble_ready(self):
+        # Expected plan after Pebble ready with default config
+        expected_plan = {
+            "services": {
+                "httpbin": {
+                    "override": "replace",
+                    "summary": "httpbin",
+                    "command": "gunicorn -b 0.0.0.0:80 httpbin:app -k gevent",
+                    "startup": "enabled",
+                    "environment": {"GUNICORN_CMD_ARGS": "--log-level info"},
+                }
+            },
+        }
+        # Simulate the container coming up and emission of pebble-ready event
+        self.harness.container_pebble_ready("httpbin")
+        # Get the plan now we've run PebbleReady
+        updated_plan = self.harness.get_container_pebble_plan("httpbin").to_dict()
+        # Check we've got the plan we expected
+        self.assertEqual(expected_plan, updated_plan)
+        # Check the service was started
+        service = self.harness.model.unit.get_container("httpbin").get_service("httpbin")
+        self.assertTrue(service.is_running())
+        # Ensure we set an ActiveStatus with no message
+        self.assertEqual(self.harness.model.unit.status, ops.ActiveStatus())
+...
+```
+
+> See more: [`ops.testing.Harness`](https://ops.readthedocs.io/en/latest/#ops.testing.Harness)
 
-        # Instantiate the charm and trigger events that Juju would on startup
-        harness.begin_with_initial_hooks()
 
-        # Update charm config and trigger config-changed
-        harness.update_config({'log_level': 'warn'})
+Explore further, start editing the files, or skip ahead and pack the charm:
 
-        # Check that charm properly handled config-changed, for example,
-        # the charm added the correct Pebble layer
-        plan = harness.get_container_pebble_plan('prometheus')
-        self.assertIn('--log.level=warn', plan.services['prometheus'].command)
 ```
+charmcraft pack
+```
+
+If you didn't take any wrong turn or simply left the charm exactly as it was, this should work and yield a file called `my-new-charm_ubuntu-22.04-amd64.charm` (the architecture bit may be different depending on your system's architecture). Use this name and the resource from the `metadata.yaml` to deploy your example charm to your local MicroK8s cloud:
+
+```
+juju deploy ./my-new-charm_ubuntu-22.04-amd64.charm --resource httpbin-image=kennethreitz/httpbin
+```
+
+Congratulations, you’ve just built your first Kubernetes charm using Ops!
```

Comment:
Everywhere else "ops" is written as `ops` so it probably should be here for consistency, although I would also be happy if it was plain "Ops" in lots of the cases as well :smile:.

```suggestion
Congratulations, you’ve just built your first Kubernetes charm using `ops`!
```

Reply:
I originally had Ops everywhere, then switched to `ops`. Happy with either! I'll stick with `ops` for now because that's what we have already.